### PR TITLE
Mobile, Tablet, and Desktop header

### PR DESCRIPTION
**Pull request by bxsmith007**

### CODE REVIEW

File: client/src/components/Header/Header.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import './Header.scss';
 import { NavLink } from 'react-router-dom';
-import { Button } from 'react-bootstrap';
+import {Button, Modal} from 'react-bootstrap';
 import SearchBar from './SearchBar/SearchBar';
+import CheeseburgerMenu from 'cheeseburger-menu'
```
Comment (njd9419) [inline client/src/components/Header/Header.js]: Minor: No `;` at the end of this `import`

File: client/src/components/Header/Header.js [hunk 2]
```diff
@@ -12,30 +13,118 @@ export default class Header extends React.Component {
...
+                                        <hr/>
+                                        <li><Button size="sm" variant="link-secondary" style={{padding: 0}}>Log in</Button></li>
+                                        <li><Button size="sm" variant="link-secondary" style={{padding: 0}}>Sign up</Button></li>
+                                    </ul>
+                                </div>
+                            </CheeseburgerMenu>
+
+                        </div>
+                    </>
+                }
```
Comment (njd9419) [inline client/src/components/Header/Header.js]: I'm not too familiar with mobile friendliness, but can't this be done via media breakpoints in `.scss` files?

Comment (njd9419) [reply client/src/components/Header/Header.js]: I believe we've done it that way for other pages.

File: client/src/components/Header/SearchBar/LocationSearch/LocationSearch.js [hunk 3]
```diff
@@ -90,7 +91,7 @@ export default class LocationSearch extends React.Component {
         };
 
         return (
-            <div className="form-control-sm p-0 position-relative">
+            <div className="form-control-sm p-0 position-relative" style={{width: width, marginRight: margin, marginLeft: margin, marginBottom: barBottomSpacing}}>
```
Comment (njd9419) [inline client/src/components/Header/SearchBar/LocationSearch/LocationSearch.js]: Again, can't this be done via CSS? This really clutters up the components.

**Comment by njd9419** [comment]
First pass completed.

A couple of questions:
1. Can't you do mobile friendliness via media breakpoints inside `.scss` files? We've done it for other pages I believe. Maybe I'm mistaken and that's not the correct way to do it though.
2. Can you move all of the `style={}` to the `.scss` files? I find it makes the components cluttered with styling that really should be in CSS files.

Also, there are conflicts.

**Comment by bxsmith007** [comment]
> First pass completed.
> 
> A couple of questions:
> 
> 1. Can't you do mobile friendliness via media breakpoints inside `.scss` files? We've done it for other pages I believe. Maybe I'm mistaken and that's not the correct way to do it though.
> 2. Can you move all of the `style={}` to the `.scss` files? I find it makes the components cluttered with styling that really should be in those separate CSS files.
> 
> Also, there are conflicts.

**Comment by bxsmith007** [comment]
So, I don't think I can do all the mobile stuff in the .scss file, because for each width difference, there are new components to potentially render. I don't think the .scss file and rerender those new components for me unless I am wrong.

I am moving as many style={{}} sections as possible to .scss files though

**Comment by benvogler** [comment]
@bxsmith007 check out https://github.com/team-monumental/monuments-and-memorials/pull/77 before you make any changes here

**Pull request closed without merging**